x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -366,7 +366,7 @@ const getSavedObjectTypes = (
     },
     mappings: {
       properties: {
-        preconfiguration_id: { type: 'keyword' },
+        id: { type: 'keyword' },
```

Comment:
Do we need to have the id in the saved object attributes? Saved object already have an id it's going to be a duplicate no?

Reply:
This is the saved object for the deletion record, not for the agent policy.